users/user-creation.md

```diff
@@ -0,0 +1,42 @@
+## User Creation
```

Comment:
This should probably be a `#` (h1) header. Likewise, the next ones should probably be `##` etc.